sdk/common/src/main/java/io/opentelemetry/sdk/common/export/ProxyOptions.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.common.export;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import java.net.ProxySelector;
+import java.net.SocketAddress;
+import java.net.URI;
+import java.util.Collections;
+import java.util.List;
+
+/** Configuration for proxy settings. */
+public final class ProxyOptions {
+  private final ProxySelector proxySelector;
+
+  private ProxyOptions(ProxySelector proxySelector) {
+    this.proxySelector = proxySelector;
+  }
+
+  /** Create proxy options with the {@code proxySelector}. */
+  public static ProxyOptions create(ProxySelector proxySelector) {
+    return new ProxyOptions(proxySelector);
+  }
+
+  /**
+   * Create proxy options with a {@link ProxySelector} which always uses an {@link Proxy.Type#HTTP}
+   * proxy with the {@code socketAddress}.
+   */
+  public static ProxyOptions create(InetSocketAddress socketAddress) {
+    return new ProxyOptions(new SimpleProxySelector(new Proxy(Proxy.Type.HTTP, socketAddress)));
+  }
+
+  /** Return the {@link ProxySelector}. */
+  public ProxySelector getProxySelector() {
+    return proxySelector;
+  }
+
+  @Override
+  public String toString() {
+    return "ProxyOptions{proxySelector=" + proxySelector + "}";
```

Comment:
Do we know that proxySelector has a proper `toString()` method?

Reply:
It will depend on the implementation, but the static inner SimpleProxySelector below does have a proper toString. We try to provide toString for all sdk components so that users can call OpentelemetrySdk#toString and understand the full config.